ldes-fragmentisers/ldes-fragmentisers-common/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/fragmentation/repository/AllocationRepository.java

```diff
@@ -10,4 +12,6 @@ public interface AllocationRepository {
 	void unallocateAllMembersFromView(ViewName viewName);
 
 	void unallocateMembersFromCollection(String collectionName);
+
+	List<String> findMembersForFragment(String fragmentId);
```

Comment:
What does this return? Based on the name, I would expect Member objects but you return Strings? I assume these are ids? Maybe add this to the name of the method?